docs/conventions.rst

```diff
@@ -0,0 +1,298 @@
+  Here :math:`\omega_b` is the angular velocity expressed in the body frame,
+  and :math:`v_b` is the linear velocity of a point at the body origin
+  (e.g. the COM) expressed in the body frame.  
+  Thus :math:`V_b = (\omega_b,\;v_b)` gives the body's own-frame view of its
+  motion.
+
+* **Spatial twist** (:math:`V_s`):
+  uses the fixed *space frame* (world/inertial frame).
+  :math:`\omega_s` is the angular velocity expressed in world coordinates, and
+  :math:`v_s` represents the linear velocity of a hypothetical point on the
+  moving body that is instantaneously at the world origin.  Equivalently,
+
+  .. math::
+
+     v_s \;=\; \dot p \;-\; \omega_s \times p,
+
+  where :math:`p` is the vector from the world origin to the body origin.
+  Hence :math:`V_s = (\omega_s,\;v_s)` is called the **spatial twist**.
+  *Note:* :math:`v_s` is **not** simply the COM velocity
+  (that would be :math:`\dot p`); it is the velocity of the *world origin* as
+  if rigidly attached to the body.
+
+In summary, *Modern Robotics* lets us express the same physical motion either
+in the body frame or in the world frame.  The angular velocity is identical
+up to coordinate rotation; the linear component depends on the chosen
+reference point (world origin vs. body origin).
+
+Physics-Engine Conventions (Drake, MuJoCo, Isaac)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Most physics engines store the **COM linear velocity** together with the
+**angular velocity** of the body, typically both in world coordinates.  This
+corresponds conceptually to a twist taken at the COM and expressed in the
+world frame, though details vary:
+
+* **Drake**  
+  Drake's multibody library uses full spatial vectors with explicit frame
+  names.  The default, :math:`V_{MB}^{E}`, reads "velocity of frame *B*
+  measured in frame *M*, expressed in frame *E*."  In normal use
+  :math:`V_{WB}^{W}` (body *B* in world *W*, expressed in *W*) contains
+  :math:`(\omega_{WB}^{W},\;v_{WB_o}^{W})`, i.e. both components in the world
+  frame.  This aligns with the usual physics-engine convention.
+
+* **MuJoCo**  
+  MuJoCo employs a *mixed-frame* format for free bodies:  
+  the linear part :math:`(v_x,v_y,v_z)` is in the world frame, while the
+  angular part :math:`(\omega_x,\omega_y,\omega_z)` is expressed in the **body
+  frame**.  The choice follows from quaternion integration (angular velocities
+  "live" in the quaternion's tangent space, a local frame).
+
+* **Isaac Lab / Isaac Gym**  
+  NVIDIA's Isaac tools provide **both** linear and angular velocities in the
+  world frame.  The root-state tensor returns
+  :math:`(v_x,v_y,v_z,\;\omega_x,\omega_y,\omega_z)` all expressed globally.
+  This matches Bullet/ODE/PhysX practice.
+
+Newton Conventions
+~~~~~~~~~~~~~~~~~~
+
+**Newton** follows the standard physics engine convention for most solvers, 
+aligning with Isaac Lab's approach, but with one important exception:
+
+* **Standard Newton solvers** (XPBD, Euler, etc.)  
```

Comment:
You can also add `MuJoCoSolver` here since it handles the `body_qd` conversion to the Newton convention.